packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -55,15 +55,15 @@ export class InsecureUrlResolver implements IUrlResolver {
             if (!fileName) {
                 throw new Error("FileName should be there!!");
             }
-            return this.resolveHelper(fileName);
+            return this.resolveHelper(fileName, "");
         }
         const parsedUrl = new URL(request.url);
-
         // If hosts match then we use the local tenant information. Otherwise we make a REST call out to the hosting
         // service using our bearer token.
         if (parsedUrl.host === window.location.host) {
             const documentId = parsedUrl.pathname.substr(1).split("/")[0];
-            return this.resolveHelper(documentId);
+            const uri = parsedUrl.pathname.replace(`/${documentId}/`,"");
```

Comment:
This isn't really a full URI, so maybe a more descriptive name (both here and as the helper param) would be something like `documentRelativePath`.